File: wheelscrawler-spa/src/app/models/SearchRequestParams.ts
import { UserParams } from "./userParams";

export class SearchRequestParams extends UserParams {
    Brand: string = '';
    Model: string = '';
    Fuel: string = '';
    Gearbox: string = '';
    Type: string = '';
    IsNeedToSave: boolean = false;

    // city: string = '';
    // engineCapacityFrom: number = 0.0;
    // engineCapacityTo = 10.0;
    // priceFrom = 0;
    // priceTo = 1_000_000;
    // kilometrageFrom = 0;
    // kilometrageTo = 1_000_000;  
    // pageNumber = 1;
    // pageSize = 21;  
}

File: WheelsCrawler.API/Helpers/SearchRequestParams.cs
using WheelsCrawler.Data.Helpers;

namespace WheelsCrawler.API.Helpers
{
    public class SearchRequestParams : UserParams
    {
        public string Brand { get; set; }
        public string Model { get; set; }
        public string Fuel { get; set; }
        public string Gearbox { get; set; }
        public string Type { get; set; }
        public bool IsNeedToSave { get; set; }

    }
}